src/devices/Pn5180/Pn5180.cs

```diff
@@ -0,0 +1,1496 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Iot.Device.Card;
+using Iot.Device.Card.Mifare;
+using Iot.Device.Rfid;
+
+namespace Iot.Device.Pn5180
+{
+    /// <summary>
+    /// A PN5180 class offering RFID and NFC functionalities. Implement the CardTransceiver class to
+    /// allow Mifare, Credit Card support
+    /// </summary>
+    public class Pn5180 : CardTransceiver, IDisposable
+    {
+        private const int TimeoutWaiting = 10_000;
+
+        private readonly SpiDevice _spiDevice;
+        private readonly GpioController _gpioController;
+        private bool _shouldDispose;
+        private int _pinBusy;
+        private int _pinNss;
+
+        private static List<SelectedPiccInformation> _activeSelected = new List<SelectedPiccInformation>();
+
+        /// <summary>
+        /// A radio Frequence configuration element size is 5 bytes
+        /// Byte 1 = Register Address
+        /// next 4 bytes = data of the register
+        /// </summary>
+        public const int RadioFrequenceConfiguraitonSize = 5;
```

Comment:
I think this should be

```suggestion
        public const int RadioFrequencyConfiguraitonSize = 5;
```
same for comment above

Reply:
This is still missspelled ;-)

Reply:
:smile: good call